agenta-backend/agenta_backend/main.py

```diff
@@ -18,7 +18,7 @@
 )
 
 if os.environ["FEATURE_FLAG"] in ["cloud", "ee"]:
-    from agenta_backend.cloud.services import templates_manager
+    from agenta_backend.commons.services import templates_manager
```

Comment:
why is templates_manager parts of commons. I thing the logic for ee and cloud is different. cloud is lambda and ee in ecr, no?

Reply:
No the logic is just to load templates from ecr instead of dockerhub.
The repo is injected with a envvar